docs/guides/social-signin/overview.mdx

```diff
@@ -0,0 +1,59 @@
+---
+id: overview
+title: Overview
+---
+
+Thanks to social sign-in, users can use their existing accounts from providers such as 
+Google, Slack, or Facebook for sign-up and sign-in in your application. 
+
+Ory Cloud provides convenient, GUI-based flows for adding new social sign-in providers through the
+Ory Console. Each flow is custom-tailored to accommodate configuration requirements of the specific 
+identity provider. 
+
+:::info
+
+This feature is being actively developed. New provider-specific flows are added to the Ory Console
+with new releases.
+
+:::
+
+By signing up with external identity providers, users give Ory access to the profile data 
+of the account created in the external identity provider. This data is used to create an Ory 
+identity and a user account in your application
+
+Allowing users to use their existing accounts with your application removes the friction
+that comes with having to remember another set of credentials and makes the decision to 
+sign up much easier.
+
+Adding social sign-in providers relies on OAuth2 authorization and the OpenID identity layer. 
+
+## OIDC Token Handling
+
+Ory Identity Service (Kratos) doesn't save the OIDC tokens it gets from social sign-in
+providers. The tokens are exchanged for Ory Sessions. The tokens are used in the following way: 
+
+1. Ory Identity Service starts the OIDC flow with a social sign-in provider.
+2. Upon successful authentication, the social sign-in provider issues an `id_token`.
+3. Ory Identity Service gets user information from the `id_token` claims.
+4. The Identity service checks if a matching identity exists. <br />
+    ✅ The user gets a Session. <br />
+    ❌ Identity Service creates a new identity and issues a Session.
+5. The system discards of the `id_token` when the flow is completed. 
+
+
+## Preventing Account Linking
+
+If you enable multiple registration methods users may be redirected to a social login provider 
+to link their existing accounts after email verification. To disable this behavior, add the `session`
+hook after one of the registration methods (for example `password`):
+
+```yaml
+selfservice:
+  flows:
+    registration:
+      after:
+        password:
+          hooks:
+            - hook: session
+```
+
```

Comment:
If we add here this section https://www.ory.sh/docs/kratos/guides/sign-in-with-github-google-facebook-linkedin#log-in-through-social-providers-if-the-identifiers-match
it will make it more 'complete' I would say.

Reply:
done